app/scripts/metamask-controller.js

```diff
@@ -1502,13 +1497,9 @@ module.exports = class MetamaskController extends EventEmitter {
    * this is a good candidate for deprecation.
    *
    * @param {*} outStream - The stream to provide public config over.
-   * @param {URL} senderUrl - The URL of requesting resource
    */
-  setupPublicConfig (outStream, senderUrl) {
-    const configStore = this.createPublicConfigStore({
-      // check the providerApprovalController's approvedOrigins
-      checkIsEnabled: () => this.providerApprovalController.shouldExposeAccounts(senderUrl.hostname),
-    })
+  setupPublicConfig (outStream) {
+    const configStore = this.createPublicConfigStore({})
```

Comment:
Nit: this empty object parameter isn't needed

Reply:
I removed the empty object in 23c5e562c